internal/telemetrygen/README.md

```diff
@@ -0,0 +1,107 @@
+# Telemetry generator for OpenTelemetry
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [alpha]: traces   |
+|               | [development]: metrics, logs   |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Acmd%2Ftelemetrygen%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Acmd%2Ftelemetrygen) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Acmd%2Ftelemetrygen%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Acmd%2Ftelemetrygen) |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@mx-psi](https://www.github.com/mx-psi), [@codeboten](https://www.github.com/codeboten) |
+
+[alpha]: https://github.com/open-telemetry/opentelemetry-collector#alpha
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+<!-- end autogenerated section -->
+
+This utility simulates a client generating **traces**, **metrics**, and **logs**. It is useful for testing and demonstration purposes.
+
+## Installing
+
+To install the latest version run the following command:
+
+```console
+go install github.com/open-telemetry/opentelemetry-collector-contrib/cmd/telemetrygen@latest
+```
+
+Check the [`go install` reference](https://go.dev/ref/mod#go-install) to install specific versions.
+
+### Docker build
+
+You can build locally the Docker image with:
+
+```bash
+make docker-telemetrygen
+```
+
+Using github actions, we also push a docker image on commit to main or on release to this [Github docker registry](https://github.com/open-telemetry/opentelemetry-collector-contrib/pkgs/container/opentelemetry-collector-contrib%2Ftelemetrygen).
+
+## Running
+
+First, you'll need an OpenTelemetry Collector to receive the telemetry data. Follow the project's instructions for a detailed setting up guide. The following configuration file should be sufficient:
```

Comment:
IIANM our use case is different than running OTEL collector, maybe we should reword this to mention any server hosting OTEL collector endpoints?

Reply:
Let me just rip the entire README and leave a note that this is forked